src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationAttribute.kt

```diff
@@ -59,14 +60,38 @@ class LocationAttribute(
   @OneToMany(mappedBy = "locationAttribute", fetch = FetchType.LAZY, cascade = [CascadeType.ALL], orphanRemoval = true)
   private val locationSchedule: MutableList<LocationSchedule> = mutableListOf()
 
-  fun setLocationSchedule(schedules: List<LocationSchedule>) {
```

Comment:
Replaced with `addSchedule( ... )`.  You would only ever be adding one entry/row to the schedule each time.